README.md

```diff
@@ -2,33 +2,23 @@
 **Available for:** Semantik Invoice
 
 ## Overview
-This repository contains helper tools that can be used for Semantik-related projects to export the necessary data from Semantik Invoice. 
+This repository contains helper tools that can be used for Semantik-related projects to export the necessary data from Semantik Invoice.
 
+The following PowerAutomate flow templates are available in this repository:
+* [Invoice to CSV](.\Docs\InvoiceToCSV.md)
+* [Invoice to pipe delimited CSV](.\Docs\PipeDelimitedFileExport.md)
+* [Invoice to pipe delimited CSV with Column Filter](.\Docs\PipeDelimitedFileExport.md)
 ## Prerequisites
-You will need access to Semantik Invoice and Microsoft PowerAutomate to complete the integration.
+You will need access to [Semantik Invoice](https://ephesoft.com/products/semantik-invoice/) and [Microsoft PowerAutomate](https://flow.microsoft.com) to complete the integration.
+### How To Get Credentials
+To procure a free Semantik Trial account or get in touch, fill out the form here: [Semantik for Invoices Trial](https://ephesoft.com/get-trial/).
 
-## Pipe delimited file Export
-This exports Semantik data into a pipe delimited file. PowerAutomate expects the following columns to be exported from Semantik in the order listed below:
-
-- Item #
-- Description
-- Shipped
-- Unit Price
-- Ext Price
-
-
-## Pipe delimited file export with column filter
-This exports Semantik data into a pipe delimited file for columns with table headers that match (case-sensitive) the expected column names. Other columns are not placed in the file.
-
-The expected column names are:
-
-- Item #
-- Description
-- Shipped
-- Unit Price
-- Ext Price
-
-**Note:** A column name with a period cannot be set as expected as it is an invalid character in Microsoft Flow expression language.
+## Importing PowerAutomate Templates
+The templates in this repository can be imported as new Microsoft PowerAutomate flows.  Follow these instructions to import a template:
+1. Create a new .zip file containing all files within the template's directory
```

Comment:
ZIP instead of .zip to follow our conventions

Reply:
Nice catch, updated.